client/ayon_core/lib/transcoding.py

```diff
@@ -362,42 +370,76 @@ def get_review_info_by_layer_name(channel_names):
         if "." in channel_name:
             layer_name, last_part = channel_name.rsplit(".", 1)
 
-        channels_by_layer_name[layer_name][channel_name] = last_part
         if last_part.lower() not in {
-            "r", "red",
-            "g", "green",
-            "b", "blue",
-            "a", "alpha"
+            # Detect RGBA channels
+            "r", "g", "b", "a",
```

Comment:
You removed `"red"`, `"green"`, `"blue"` and `"alpha"`.

Reply:
That's correct - but if you check the old code.. it'd have never been used. Because [at line 389-391](https://github.com/ynput/ayon-core/pull/1271/files#diff-6186b91d85fadce30ee10096d4d231bd7feb578fb64d977f6335eeb6dc55073cL387-L392) it would only ever have used R, G, B. And since we stored them as `.upper()` it should've then also checked `RED`, `GREEN`, `BLUE` keys, right?

Reply:
☝️